modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -106,6 +106,7 @@
             <class name="org.wso2.identity.integration.test.saml.SAMLInvalidIssuerTestCase"/>
             <class name="org.wso2.identity.integration.test.saml.RegistryMountTestCase"/>
             <class name="org.wso2.identity.integration.test.provisioning.JustInTimeProvisioningTestCase"/>
+            <class name="org.wso2.identity.integration.test.security.SECINT115EntitlementXSSTestCase"/>
```

Comment:
You have placed this under <!--SAML Test Cases--> , Shall we move this to <!--Entitlement Test Cases--> or to a new category <!--SecurityTest Cases-->